dotCMS/src/main/java/com/dotmarketing/portlets/structure/business/FieldAPIImpl.java

```diff
@@ -56,6 +56,13 @@ public boolean isElementConstant(Field field) {
 		}
 		return false;
 	}
+
+	public boolean isElementHidden(Field field) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Field"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)